.circleci/config.yml

```diff
@@ -593,54 +558,54 @@ jobs:
         default: []
     environment:
       WORKBENCH_ENV: << parameters.env_name >>
-      TEST_MODE: << parameters.test_mode >>
     parallelism: << parameters.parallel_num >>
     steps:
-      - checkout-code
+      - attach_workspace:
+          at: .
       - steps: << parameters.optional_steps >>
-      - halt-test-check:
-          dir_names: "api ui e2e"
       - run:
-          name: Update test user environment variables
-          command: |
-            case << parameters.env_name >> in
-              "staging")
-                echo 'export STAGING_COLLABORATOR=$PUPPETEER_COLLABORATOR_STAGING' >> $BASH_ENV
-                echo 'export USER_NAME=$PUPPETEER_USER_STAGING@staging.fake-research-aou.org' >> $BASH_ENV
-                ;;
-              "test")
-                echo 'export TEST_COLLABORATOR=$PUPPETEER_COLLABORATOR_TEST' >> $BASH_ENV
-                echo 'export USER_NAME=$PUPPETEER_USER_TEST@fake-research-aou.org' >> $BASH_ENV
-                ;;
-              "local")
-                echo 'export DEV_COLLABORATOR=$PUPPETEER_COLLABORATOR_LOCAL' >> $BASH_ENV
-                echo 'export USER_NAME=$PUPPETEER_USER_LOCAL@fake-research-aou.org' >> $BASH_ENV
-                ;;
-            esac
-            echo 'export PASSWORD=$PUPPETEER_USER_PASSWORD' >> $BASH_ENV
-            source $BASH_ENV
+          name: "Update environment variables for e2e test"
+          command: bash .circleci/e2e-test-users.sh << parameters.env_name >>
```

Comment:
I would put a verb into this script name, maybe `export-e2e-test-user-vars.sh`, or `e2e-export-user-vars.sh`

Reply:
sounds good. renamed to `export-e2e-test-user-vars.sh`